File: 2006-count-number-of-pairs-with-absolute-difference-k/2006-count-number-of-pairs-with-absolute-difference-k.c


int countKDifference(int* nums, int numsSize, int k){

    int total = 0;
    
    for (int i = 0; i < numsSize-1; i++){
        
        for(int j = i; j < numsSize; j++){
            if((nums[i] - nums[j] == k) || (nums[i] - nums[j] == -k))
                total ++;
        }
    }
    return total;
}

File: 2006-count-number-of-pairs-with-absolute-difference-k/2006-count-number-of-pairs-with-absolute-difference-k.java
class Solution {
    public int countKDifference(int[] nums, int k) {
        int control;
        int total = 0;
        for (int i = 0; i < nums.length-1;i++){
            for (int j = i+1; j < nums.length;j++){
                control = Math.abs(nums[i]-nums[j]);
                if (control == k){
                    total +=1;
                }
            }
        }
        return total;  
    }
}